tools/tf2openapi/cmd/main.go

```diff
@@ -13,6 +13,7 @@ import (
 var (
 	model   = flag.String("model", "", "Absolute path of SavedModel file")
 	outFile = flag.String("output-file", "", "Absolute path of file to write OpenAPI spec to")
+	sigDefKey = flag.String("signature-def", "", "Serving Signature Def Key ")
```

Comment:
We need to document these CLI options at some point.

Reply:
Got it. Will open a separate PR for that

Reply:
https://github.com/kubeflow/kfserving/pull/220/commits/58e84f67ffbd386e9d7ce1bf31b5d36d02e52bd1